app.js

```diff
@@ -0,0 +1,127 @@
+const statusDisplay = document.querySelector('.game--status');
+const board = document.querySelector('.board');
+const form = document.getElementById('form');
+const alert = document.querySelector('.alert');
+let gameActive = true;
+
+function Player(name, symbol) {
+  this.name = name;
+  this.symbol = symbol;
+}
+
+const playerOne = new Player('', 'X');
+const playerTwo = new Player('', 'O');
+
+
+const currentPlayerTurn = (cp) => `It's ${cp}'s turn`;
+
+let gameState = ['', '', '', '', '', '', '', '', ''];
```

Comment:
- [ ] **[OPTIONAL]** We can use array class initialization to have more idiomatic, less repititive code: 

```js
let gameState = new Array(9).fill('');